apps/web/components/apps/App.tsx

```diff
@@ -441,7 +443,16 @@ const InstallAppButtonChild = ({
         </Button>
       </DropdownMenuTrigger>
       <DropdownMenuPortal>
-        <DropdownMenuContent>
+        <DropdownMenuContent
+        onInteractOutside={(event) => {
+          if (mutation.isLoading) event.preventDefault();
+        }}
```

Comment:
This prevents the user from closing the dropdown menu until adding the app resolves with a success + redirect or an error message